### DISCUSSION TITLE
IE 11 and Chrome Version 81 Issues with button dropdown

### DISCUSSION THREAD

**Discussion by morencigirl** [twbs/bootstrap#30833] [General]
Using this in IE 11 and and Chrome 81, cannot get the dropdown to work:

`<div class="btn-group-vertical">`
`<div class="btn-group-vertical" role="group" aria-label="Button group with nested dropdown">`
 `<button id="btnGroupDrop1" type="button" class="btn btn-light dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Home</button>`
`<div class="dropdown-menu" aria-labelledby="btnGroupDrop1">` 
`<a class="dropdown-item" href="#">Home</a>`
  `</div>`
  `</div>`

Link won't work in IE but does in Chrome.
`<button type="button" class="btn btn-light" href="/sites/yyy/xxx/Basic_JS_File.aspx">Basic JS File</button>`

Any ideas?

**Comment by ffoodd** [comment]
Please open an issue with a reduced test case (please read and follow the issue template).

FWIW, your second code block is useless: `href` is for links (so either `<a>`  or `<link>`).

**Reply by morencigirl** [reply]
Thanks.  Is this a question or a bug?

**Comment by mdo** [comment]
Without knowing more about your setup, this likely has something to do with how you've setup Bootstrap and your project. Make sure the JS is included, your HTML correct, etc.
